src/prompt.c

```diff
@@ -574,6 +574,11 @@ prompt_menu(const char *prompt, const struct menu_item *items, int *selected)
 			break;
 
 		default:
+			if (key.modifiers.control && (key.data.bytes[0] == 'C')) {
```

Comment:
The `multibytes` flag was added to support UTF-8 keybindings. Before Tig simply read the `int` returned by `wgetch`. `multibytes` is set if when ncurses returns one of its internal key codes.

> The curses wgetch() function
> will return any given one of these only if the corresponding k- capability
> is defined in your terminal's terminfo entry.

Maybe they no longer apply with the use of raw mode?